interface/src/components/gmap.js

```diff
@@ -1,62 +1,166 @@
 import React, { Component } from 'react'
-import { Map, GeoJSON, TileLayer } from 'react-leaflet'
-import { getGeoJSON } from './requests'
-import './stylesheets/leaflet.css'
-import { bbox } from '@turf/turf'
+import { Map, Popup, GeoJSON, TileLayer } from 'react-leaflet'
+import { withRouter, Redirect } from 'react-router-dom'
+import getRandomColors from './getRandomColor'
+import * as d3 from 'd3';
+import { getGeoJSON, getVisualData } from './requests'
+import './stylesheets/map.css'
+import { bbox, geojsonType } from '@turf/turf'
 
 
 class GMap extends Component{    
     constructor(props) {
         super(props);
         this.state={
-            position: [51.505, -0.09]
-      }    
+            position: [ 33.056702, -112.046656 ],
```

Comment:
Can `position` be a prop? It could also be stored in a cookie, HTML5 indexedDB (https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API), or other storage, instead of being hard coded to this spot.

Reply:
Yeah, we can try this. Actually, it's the default position [ 33.056702, -112.046656 ] for the map pointing to Maricopa, AZ.  Within a few seconds, the map displays the shapefile data so do we really need to make the user location as default?